packages/flutter/test/widgets/semantics_traversal_test.dart

```diff
@@ -202,13 +202,13 @@ void main() {
     await tester.test(
       textDirection: TextDirection.ltr,
       children: children,
-      expectedTraversal: 'A B C D E F G H I J K L M',
+      expectedTraversal: 'A B C D E G H F I J K L M',
```

Comment:
This is ... odd

Reply:
Yes, I think something's wrong here.

Unfortunately, the regression in semantics_tester was introduced in the same commit where these tests were added.  I tried checking out d35409629994769a8ce26cad4651e20ab07280d7 and modifying the tester so that it's closer to the state in this PR and those tests still pass.  I'll try to figure out what's going on here, unless @yjbanov might know.

Reply:
Looks like this regressed in https://github.com/flutter/flutter/commit/f2442b8a13b4100a0734aa4cbf30bf0ff89ce5fd, which was part of v0.8.0 tag.

Reply:
(And wasn't caught because the tester was broken)

Reply:
🤕

Reply:
Got this sorted out - changed the deflate constant from 0.1 to 1e-15.  Anything bigger than 1e-15 was failing.

Reply:
Pushed this back to .1.  The right fix is to update the traversal logic.